app/views/users/search.html.erb

```diff
@@ -1,42 +1,51 @@
-<div class="jumbotron">
-    <h2><%= I18n.t('user.lists') %></h2>
+<%= stylesheet_link_tag 'users', media: 'all', 'data-turbolinks-track': 'reload' %>
 
-      <%# https://getbootstrap.com/docs/5.0/layout/grid/#auto-layout-columns %>
+<div class="stack">
+  <%= form_tag(search_users_path, method: "get", class: "form-tag justify-content-between") do %>
+    <%= text_field_tag :search, params[:search] , placeholder: I18n.t('user.search.place_holder'), class: "text-field form-control my-2 my-sm-0"%>
+    <%= submit_tag I18n.t('user.search.index') , class: "submit-button btn btn-outline-secondary my-2 my-sm-0"%>
+  <% end %>
 
-
-    <div class="row">
-        <div class="col">
-          <%= form_tag(search_users_path, method: "get") do %>
-            <%= label_tag(:search, I18n.t('user.search.for') ) %>
-            <%= text_field_tag :search, params[:search] %>
-            <%= submit_tag I18n.t('user.search.index') %>
+  <% @contacts.each do |user| %>
```

Comment:
is there a reason, why the logic is changed? should I first see my contacts and later other people?

when #158  ist merged you can just call `<%= render partial: 'users/card', locals: { user: contact, mode: :overview } %>` here to get the expected card with the social accounts

Reply:
> is there a reason, why the logic is changed? should I first see my contacts and later other people?

Yes, that is the desired behavior.

Reply:
I would like to use the new design, especially since it is a pretty nice generic solution. But after some testing I am a little bit conflicted:  

<img width="372" alt="Screenshot 2021-02-03 at 20 59 19" src="https://user-images.githubusercontent.com/32880898/106802171-dddcdf00-6662-11eb-926e-94580bc22b7a.png">

I just changed the code to:

```rails
      <% @users.each do |user| %>
             <%= render partial: 'users/card', locals: { user: user, mode: :overview } %>
     <% end %>
```

Am I doing something wrong? 😞

Reply:
If you can't find a quick fix I could try to adjust it with some CSS.

Reply:
I dont think you are doing somethin wrong and for me with firefox and iPhone 6/7/8 it looks like this:
![grafik](https://user-images.githubusercontent.com/49535291/106805448-fbac4300-6666-11eb-9ec0-7c13357eb9c3.png)

Reply:
I changed the code, to the desired partial but I am on your branch

Reply:
@Paula-Kli there are a few problems with the cards that catch my eye:

* Why are we using tables to style the cards? Tables were used in the early days of CSS. Now usually flex or grids are used for that. With flex, you are able to place the content much nicer and evenly. For example, you can distribute the handles evenly. I think the problem that we saw is caused by using tables.
* Why are we using cards from Bootstrap. Bootstrap cards usually have a different layout. If we want to simply display a shadow and have round corners we can  use basic CSS

I could quickly adjust the code of your partials so that the problems are solved if it is ok for you 🙃 . We already have all the CSS.

Reply:
will be solved with #189 .